cmd/gotk/get_source_helm.go

```diff
@@ -49,36 +52,39 @@ func getSourceHelmCmdRun(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"Name", "Revision", "Ready Status", "Message", "Namespace"})
+	table.SetRowLine(true)
+
 	var list sourcev1.HelmRepositoryList
 	err = kubeClient.List(ctx, &list, client.InNamespace(namespace))
 	if err != nil {
 		return err
 	}
 
 	if len(list.Items) == 0 {
-		logger.Failuref("no sources found in %s namespace", namespace)
+		table.Append([]string{"None", "None", "Unknown", "No sources found", namespace})
+		table.Render()
```

Comment:
If there are no sources found, this should not print a table but simply log a string to `stderr` (as it did before the change).

The printed string could be more informative about the resource type, e.g. `no helm sources found in %s namespace`.